### ISSUE TITLE
Add sendgrid kinesis to AWS developer tools

### ISSUE THREAD

**Issue by jasonrichardsmith** [sendgridlabs/go-kinesis#52]
This package should be listed here.

http://aws.amazon.com/kinesis/streams/developer-resources/

which can be done by posting here:
https://forums.aws.amazon.com/forum.jspa?forumID=169&start=0

**Comment by le0pard** [comment]
Right now I don't have time to create post on AWS forum. As you wish, you can do it. Thanks in advance.

**Comment by pavelnikolov** [comment]
@jasonrichardsmith I don't think this library is production ready. One important feature that is missing is synchronizing shard iterators in a multi-instance app (ensuring that different instances do not process duplicate data).

**Comment by jasonrichardsmith** [comment]
I have fallen back to the KCL at this point because of the points you illustrated.
